lib/caps/DG2/info

```diff
@@ -10,4 +10,5 @@
 
 info = dict(
   gpu = dict(gen = 12.7),
+  stringapi = dict(enable = True),
```

Comment:
Also seems out of place.  This gives the impression that string api will only work on specific platforms.

Reply:
> Also seems out of place. This gives the impression that string api will only work on specific platforms.
the stringapi is only enabling on libvpl, so there are platforms limited, so added platform caps define.
do you have any suggestion?